src/Portfolio-menu.js

```diff
@@ -1,20 +1,23 @@
-import React from 'react';
+import React from "react";
 
-import './portfolio-menu.styles.scss';
+import "./portfolio-menu.styles.scss";
 
 const PortfolioItem = ({ imageUrl, size, summary, subtitle }) => (
   <div className={`${size} portfolio-item`}>
     <div
-      className='background-image'
+      className="background-image"
       style={{
         backgroundImage: `url(${imageUrl})`
       }}
     />
-    <div className='content'>
-    <h1 className='title'>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;{summary}<br></br></h1>
-    <p className='subtitle'>{subtitle}</p>
-    </div> 
+    <div className="content">
+      <h1 className="title">
+        <span styles={{ marginLeft: "40px" }}>{summary}</span>
```

Comment:
Likewise use margins instead of spaces. Try not to put styles inline like I have above if you're using css, consistency is king